src/PatchOrchestrationApplication/NodeAgentService/src/InstallAndRunService.bat

```diff
@@ -52,6 +52,7 @@ REM icacls %workingDir% /grant "Network Service":(OI)(CI)F /T
 
 REM Stop the service and uninstall the current version
 sc stop %serviceName%
+powershell -File "KillService.ps1"
```

Comment:
Need to ensure, that restrictive execution policy doesnt block running this script.. Since this is powershell. Assuming this script would be signed so should be ok.